service/version_bundles.go

```diff
@@ -362,9 +362,9 @@ func NewVersionBundles() []versionbundle.Bundle {
 			Dependencies: []versionbundle.Dependency{},
 			Deprecated:   false,
 			Name:         "aws-operator",
-			Time:         time.Date(2018, time.February, 6, 12, 17, 0, 0, time.UTC),
+			Time:         time.Date(2018, time.February, 9, 17, 10, 0, 0, time.UTC),
```

Comment:
The time is fix. It is the creation time, not the "we developed something and now it is not in development mode anymore time".